docs/geometry.rst

```diff
@@ -265,3 +265,26 @@ single tile.
    .. automethod:: inspect
 
    .. automethod:: compare
+
+Generic Detector
+----------------
+
+Generic detector may represent an unconstrained detector of your choice.
+
+.. autoclass:: GenericGeometry
+
+   .. automethod:: from_simple_description
+
+   .. automethod:: compare
+
+   .. automethod:: get_pixel_positions
+
+   .. automethod:: inspect
+
+   .. automethod:: output_array_for_position_fast
+
+   .. automethod:: plot_data_fast
+
+   .. automethod:: position_modules_fast
+
+   .. automethod:: write_crystfel_geom
```

Comment:
This is definitely me being picky, but can I ask you to order the methods similarly to the other detectors above?

I've roughly tried to arrange methods like this:

- Classmethods to create a geometry object (`from_*`)
- Methods to convert it into another format, either in memory (`get_pixel_positions` / `to_distortion_array`) or writing files
- Methods for working with image data
- Inspection (`inspect` / `compare`)

Reply:
Sure, no problem. (I've put `from_*` in the beginning, and then just alphabetically.) Done